src/main/java/duke/Task.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+/**
+ * Represents a task given by the user, containing the name and the completion status of the task.
+ */
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+    
+    /**
+     * Creates a new task with the specified description.
+     * @param description The description of the task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Gets the string of the completion status of the task.
+     * @return A string of the status icon.
```

Comment:
Perhaps you should leave a line between your description and your @return line?

Reply:
Alright thanks